src/components/TopNav/MobileNav.js

```diff
@@ -15,7 +15,7 @@ const MobileNav = ({ showLeftMenu, onClickLeftMenu, logo, rightMenu }) => (
         )}
       </button>
     </div>
-    <a href='/'>
+    <a href={loggedIn ? '/my-dashboard' : '/'}>
```

Comment:
Please don't hardcoded the URL, put this inside config file.

Use URL.BASE to non logged user and add a new config to logged user.